src/Codeception/Command/DryRun.php

```diff
@@ -93,7 +93,7 @@ public function execute(InputInterface $input, OutputInterface $output): int
         $eventDispatcher->dispatch(new SuiteEvent($suiteManager->getSuite(), null, $settings), Events::SUITE_BEFORE);
 
         foreach ($tests as $test) {
-            if ($test instanceof PHPUnitDataProvider) {
+            if ($test instanceof DataProviderTestSuite) {
```

Comment:
I saw the opinion of **DavertMik** about it, but in `5.0` I think it is better to remove it... According https://github.com/Codeception/Codeception/pull/6136#discussion_r589233283.

Reply:
I would rather remove it from Codeception 4 because it counts as a new feature now and it could be a breaking change.

Reply:
Dead DataProviderTestSuite conditions were removed from 4.1 by #6138 ,
This PR will remove them from 5.0